tests/integration/test_charm.py

```diff
@@ -52,11 +60,11 @@ async def test_seldon_deployment(ops_test: OpsTest):
     )
 
     with open("examples/serve-simple-v1.yaml") as f:
-        sdep = SeldonDeployment(yaml.safe_load(f.read()))
+        sdep = seldon_deployment(yaml.safe_load(f.read()))
         client.create(sdep, namespace=namespace)
 
     for i in range(30):
-        dep = client.get(SeldonDeployment, "seldon-model", namespace=namespace)
+        dep = client.get(seldon_deployment, "seldon-model", namespace=namespace)
```

Comment:
It was this way when you got here, but can we update this to use something like [tenacity](https://github.com/canonical/katib-operators/blob/479d003528e9a02d07909a1a2062c549180ef72c/tests/integration/test_charms.py#L128)?  We've settled on that sometime recently as the way we persistently try things

Reply:
Done.